bundler/lib/bundler/lockfile_parser.rb

```diff
@@ -94,7 +94,7 @@ def self.bundled_with
       lockfile_contents.split(BUNDLED).last.strip
     end
 
-    def initialize(lockfile)
+    def initialize(lockfile, validate_dependencies: false)
```

Comment:
I wonder if this should be `**materialization_options`, instead. Since it's not really used by this class, but just passed around directly to `LazySpecification#initialize`, I wonder if that approach represents that better.

Reply:
Makes sense to me.